### ISSUE TITLE
WIFI 2.4 - MediaTek MT7603E 802.11bgn on xiaomi mi3g v1

### ISSUE THREAD

**Issue by FRM-CCA** [openwrt/mt76#381]
WIFI 2.4 - MediaTek MT7603E 802.11bgn on xiaomi mi3g v1

**Comment by Openwrtfunboy** [comment]
Can you give more information?

**Comment by FRM-CCA** [comment]
No, sorry i'm not expert in linux, so, since update WIFI bgn not start, no network (but i had the same problem 1 or 2 years ago)
So if you had proc, to give you more infos
No problem on 5g wifi

**Comment by ar-nim** [comment]
Try setting the channel manually, does it work?
For me when I set my channel to auto it refuses to work

**Comment by Bartvz** [comment]
@FRM-CCA try factory resetting it. I had the same problem but that fixed it. Has to do with different naming for wireless PHY's.
